HotelBooking.Data/Extensions/AppExtension.cs

```diff
@@ -0,0 +1,23 @@
+﻿using HotelBooking.Data.DataSeeder;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace HotelBooking.Data.Extensions
+{
+    public static class AppExtension
+    {
+        public static IApplicationBuilder InitDb(this IApplicationBuilder app)
```

Comment:
Why do we need this InitDb method?

Reply:
Because I want to create two roles:  Amin and Customer and a SuperUser in Admin Role first. Once the app runs, It will auto-initial them for me if they don't exist in the database.

Reply:
then makes sense for me